gameplay/src/Matrix.cpp

```diff
@@ -906,12 +906,12 @@ void Matrix::transformPoint(const Vector3& point, Vector3* dst) const
 void Matrix::transformVector(Vector3* vector) const
 {
     GP_ASSERT(vector);
-    transformVector(vector->x, vector->y, vector->z, 0.0f, vector);
+    transformVector(vector->x, vector->y, vector->z, 1.0f, vector);
```

Comment:
Vector transformation works like matrix transformation. The translation component of a matrix is Vector4(x, y, z, 1.0f). If the the w value is 0.0f the result is incorrect.